pkg/identityserver/store/migrations/20241001000000_email_notification_preferences.down.sql

```diff
@@ -0,0 +1,15 @@
+ALTER TABLE users DROP COLUMN IF EXISTS email_notification_preferences CASCADE;
```

Comment:
I don't think the `CASCADE` is necessary here but I also believe it doesn't impact the migration since its not being referenced as a foreign key anywhere.